src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -98,7 +99,18 @@ internal static class GoToDefinitionHelpers
         public static bool TryGoToDefinition(
             ImmutableArray<INavigableItem> navigableItems,
             string title,
-            Workspace workspace,
+            Workspace originalWorkspace,
```

Comment:
there is only 1 workspace, so why it is original workspace, what is other workspace?